docs/static-pvc.md

```diff
@@ -0,0 +1,115 @@
+# Static PVC with ceph-csi
+
+- [Static PVC with ceph-csi](#static-pvc-with-ceph-csi)
+  - [RBD static PVC](#rbd-static-pvc)
+    - [Create RBD image](#create-rbd-image)
+    - [Create RBD static PV](#create-rbd-static-pv)
+    - [RBD Volume Attributes in PV](#rbd-volume-attributes-in-pv)
+    - [Create RBD static PVC](#create-rbd-static-pvc)
+
+This document outlines how to create static PV and static PVC from existing rbd image
+
+**warning** static PVC can be created, deleted, mounted and unmounted but
```

Comment:
Nit: while not an immediate concern, I can see the use for being able to clone a static PV into a dynamic PV for the purposes of recovery. i.e. you are using static PVs because you had some type of failure and you need access to your pre-existing PVs again. If we don't offer a path to get out of this state, they will forever be stuck as static PVs.

Reply:
yes, Jason, I agree, once we add support for other operation like a clone, etc. we can remove this warning just added here so that users should be aware of it for now